engine/src/test/java/org/camunda/bpm/engine/test/bpmn/event/timer/StartTimerEventTest.java

```diff
@@ -1676,4 +1804,16 @@ private void cleanDB() {
     commandExecutor.execute(new DeleteJobsCmd(jobId, true));
   }
 
+  protected void createAndDeployProcessWithStartTimer() {
+    BpmnModelInstance modelInstance = Bpmn.createExecutableProcess("process")
+        .startEvent()
+        .timerWithCycle("#{myBean.getCycle()}")
+        .userTask("aTaskName")
+        .endEvent()
+        .done();
+
+    testRule.deploy(repositoryService.createDeployment()
+        .addModelInstance("process.bpmn", modelInstance));
```

Comment:
🔧 I think you can deploy the model instance directly using the test rule if I am not mistaken. But maybe I am missing something here and you're doing this on purpose 🙂
```suggestion
    testRule.deploy(Bpmn.createExecutableProcess("process")
        .startEvent()
        .timerWithCycle("#{myBean.getCycle()}")
        .userTask("aTaskName")
        .endEvent()
        .done());
```